frontend/docs/admin-guide/branding.md

```diff
@@ -8,55 +8,48 @@ Use the guide below to customize MLSpace to reflect your organization's unique b
-`APPLICATION_NAME` variable in the 'env.js' file. This value is generated during deployment and is updated in the
-`constants.ts` in the CDK project files. By default, the value is "MLSpace".
+Easily change the MLSpace name to align with your program or organization. This name is set in one place using the `APPLICATION_NAME` variable in the 'env.js' file. This value is generated during deployment and is updated in the `constants.ts` file in the CDK project files. By default, the value is "MLSpace."
 
-When the `APPLICATION_NAME` is changed, the new name will appear to users everywhere MLSpace is used in the front-end
-of the application. The updated name will not be reflected in any backend resources and no references to "MLSpace"
-in the source code will be changed.
+When the `APPLICATION_NAME` is changed, the new name will appear to users everywhere MLSpace is used in the front-end of the application. The updated name will not be reflected in any backend resources, and no references to "MLSpace" in the source code will be changed.
 
-In the example below, `APPLICATION_NAME` was set to "Demo Space".
+In the example below, `APPLICATION_NAME` was set to "Demo Space."
 
 ![Demo Space Branding](../img/branding/demospace-branding.png)
 
 ## Logo Replacement
 
 Keeping the default MLSpace logo has the following advantages:
-- It dynamically changes color with theme changes
-- It satisfies accessibility criteria
-- The SVG is part of the bundle and will have to be replaced again after major updates
+- It dynamically changes color with theme changes.
+- It satisfies accessibility criteria.
+- The SVG is part of the bundle and will have to be replaced again after major updates.
 
-It is possible to replace the logo by replacing the SVG in `shared/layout/logo/logo.tsx` file in the front-end code.
+It is possible to replace the logo by replacing the SVG in the `shared/layout/logo/logo.tsx` file in the front-end code.
 
-If the logo is replaced with a new single-colored SVG, replacing the fill of the SVG drawing elements with 'currentColor'
-will allow it to adapt to the current theme by taking on the current font color of the side-navigation. This will
-only work if the new SVG logo is a single-colored SVG as only the 'currentColor' changes with the mode swap.
+If the logo is replaced with a new single-colored SVG, replacing the fill of the SVG drawing elements with 'currentColor' will allow it to adapt to the current theme by taking on the current font color of the side-navigation. This will only work if the new SVG logo is a single-colored SVG, as only the 'currentColor' changes with the mode swap.
 
 The logo SVG can also be replaced with an image if desired.
 
 ### Light or Dark Mode Alternate Logos
 
-To return different images or SVGs based on the current user mode preference the following can be used when
-returning a logo in `shared/layout/logo/logo.tsx`:
```

Comment:
I've noticed in several places of the documentation that we have random newlines like this. Is this a relic of copy/pasting the docs from somewhere else or did someone make the conscious decision to do this? Does it affect the look?

Reply:
It doesn't affect the look. I think people were putting in line breaks so it wasn't one long line. The LLM preferred them on oneline and adding back in line breaks didn't feel necessary.